types/core-js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for core-js 0.9
+// Type definitions for core-js 0.9.1
```

Comment:
We don't parse a patch version here, since the published patch version is automatically incremented on every change.

Reply:
What am I supposed to do here?

Reply:
I've updated it with a minor version bump